composer.json

```diff
@@ -16,15 +16,15 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "ircmaxell/password-compat": "~1.0",
-        "paragonie/random_compat": "~1.0|~2.0|~9.99",
+        "php": ">=7.2",
         "symfony/intl": "~2.3|~3.0|~4.0|~5.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "~3.2"
+        "symfony/phpunit-bridge": "^4.4"
```

Comment:
this should even be `^4.4 || ^5.0` (or maybe only `^5.0` if that's fine for the min PHP version), to match the way we use the bridge in Symfony itself: we use the latest version even when testing older branches (which is why `symfony/phpunit-bridge` has a separate policy for the PHP support)

Reply:
updated